test/tests/materials/yttria/yittra_only_engineering_scale.i

```diff
@@ -12,78 +12,78 @@
 []
 
 [Variables]
-  [./temperature]
+  [temperature]
     initial_condition = 300.0
-  [../]
+  []
 []
 
 [Kernels]
-  [./HeatDiff]
+  [HeatDiff]
     type = HeatConduction
     variable = temperature
     diffusion_coefficient = yttria_thermal_conductivity
-  [../]
-  [./HeatTdot]
+  []
+  [HeatTdot]
     type = SpecificHeatConductionTimeDerivative
     variable = temperature
     specific_heat = yttria_specific_heat_capacity
     density = yttria_density
-  [../]
+  []
 []
 
 [BCs]
-  [./external_surface]
+  [external_surface]
     type = InfiniteCylinderRadiativeBC
     boundary = right
     variable = temperature
     Tinfinity = 300
     boundary_radius = 0.01
     cylinder_radius = 1
     boundary_emissivity = 0.9
-  [../]
-  [./top_surface]
+  []
+  [top_surface]
     type = FunctionDirichletBC
     boundary = top
     variable = temperature
     function = '293 + 100.0/60.*t' #stand-in for the 100C/min heating rate
-  [../]
+  []
 []
 
 [Materials]
-  [./yttria_thermal_conductivity]
+  [yttria_thermal_conductivity]
     type = ParsedMaterial
     coupled_variables = 'temperature'
     expression = '3214.46 / (temperature - 147.73)' #in W/(m-K)
     property_name = 'yttria_thermal_conductivity'
     output_properties = yttria_thermal_conductivity
     outputs = 'csv exodus'
-  [../]
-  [./yttria_specific_heat_capacity]
+  []
+  [yttria_specific_heat_capacity]
     type = DerivativeParsedMaterial
     property_name = yttria_specific_heat_capacity
     coupled_variables = 'temperature'
-    constant_names =        'molar_mass    gtokg'
-    constant_expressions =  '225.81         1.0e3' #
+    constant_names = 'molar_mass    gtokg'
+    constant_expressions = '225.81         1.0e3' #
```

Comment:
Please correct the spacing here to maintain the vertical alignment between the constant names and the values